tests/linux_env.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sudo apt-get install xdotool
```

Comment:
Additionally we need here:
rclone  
git 
p7zip-full

Reply:
I assume git will be installed to clone our code, so it it unnecessary here. (other two are added)